.github/workflows/test.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
       - run: docker pull consensys/teku:latest
-      - run: go test -timeout=5m github.com/obolnetwork/charon/app -integration -slow
+      - run: go test -timeout=5m -v github.com/obolnetwork/charon/app -integration -slow
```

Comment:
discussion: any reason why we need verbose logs?

Reply:
should remove sorry, it was failing so was just for deubbing.